src/display/editor/tools.js

```diff
@@ -1190,13 +1195,9 @@ class AnnotationEditorUIManager {
   }
 
   #addDragAndDropListeners() {
-    document.addEventListener("dragover", this.#boundDragOver);
-    document.addEventListener("drop", this.#boundDrop);
-  }
-
-  #removeDragAndDropListeners() {
-    document.removeEventListener("dragover", this.#boundDragOver);
-    document.removeEventListener("drop", this.#boundDrop);
+    const signal = this._signal;
+    document.addEventListener("dragover", this.#boundDragOver, { signal });
+    document.addEventListener("drop", this.#boundDrop, { signal });
```

Comment:
Could we bind the listener functions inline, since we no longer need to remove them manually?
```suggestion
    document.addEventListener("dragover", this.dragOver.bind(this), { signal });
    document.addEventListener("drop", this.drop.bind(this), { signal });
```